docs/UserGuide.md

```diff
@@ -156,37 +182,85 @@ Format: `exit`
-If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
+If your changes to the data file makes its format invalid, RC4HDB will discard all data and start with an empty data file at the next run.
 </div>
 
+### Importing from CSV file : `import`
+
+RC4HDB has the ability to import data through .csv files. In order for RC4HDB to find your files, place them in the data directory, `[JAR file location]/data`.
+
+Format: `import FILENAME`
+
+:information_source: The csv file that you want to have imported must follow this [format](#csv-file-format).<br>
+
+Examples:
+* `import students.csv`
+* `import residents.csv`
+
+### Exporting to CSV file : `export`
+
+RC4HDB has the ability to export data to .csv files. The file will be safe to remove from the directory, `[JAR file location]/data`.
+
+Format: `export FILENAME`
+
+:information_source: The csv file will be exported in this [format](#csv-file-format).<br>
+
+Examples:
+* `export students.csv` will export the current data file into a csv file named `students.csv`
+
+### CSV file format
+
+| INDEX 	| NAME 	| PHONE NUMBER 	|    EMAIL      | FLOOR 	| UNIT NUMBER 	| TAGS 	|
+|-------	|------	|--------------	| ------        | -------	|-------------	|------	|
+| 1      	| John Doe | 91234567  	| johnDoe@gmail.com       | 05      	|  08         	| Male 	|
+| 2      	| Maggie Smith	| 98765432	| maggieSmith@gmail.com        | 04   	|  01         	| Female|
+
+<br>
+
+<!---
 ### Archiving data files `[coming in v2.0]`
 
-_Details coming soon ..._
+_Details coming soon ..._ --->
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: First, export the data from your computer using the export command and transfer the file to the new computer. Install RC4HDB on the other computer and use the import command to import the data from the csv file.
+
+**Q**: Can I search using fields other than the name?<br>
+**A**: You can use the filter command to search for people using the other fields.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command summary
 
 Action | Format, Examples
 --------|------------------
+**Help** | `help`
 **Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
+**List** | `list`
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
```

Comment:
Update this to `edit MATRIC_NO n/NAME p/PHONE_NUMBER e/EMAIL f/FLOOR u/UNIT g/GENDER h/HOUSE mn/MATRIC_NO ma/MAJOR [t/TAG]...`